docusaurus/docs/adding-bootstrap.md

```diff
@@ -29,7 +29,7 @@ import 'bootstrap/dist/css/bootstrap.css';
 
 > Note: this feature is available with `react-scripts@2.0.0` and higher.
 
-Sometimes you might need to tweak the visual styles of Bootstrap (or equivalent package).<br>
+Sometimes you might need to tweak the visual styles of Bootstrap (or equivalent package).<br />
```

Comment:
Off-topic, but I think all these `<br/>` could be either removed or we add an empty line separating the two blocks. There isn't really a need for `<br/>`s in practice.

Reply:
👍 removing and adding an empty line sounds good to me